builtInFunctions/logsAndEvents.go

```diff
@@ -14,6 +14,31 @@ const (
 	esdtRandomSequenceLength = 6
 )
 
+func addESDTEntryForTransferInVMOutput(
+	vmInput *vmcommon.ContractCallInput,
+	vmOutput *vmcommon.VMOutput,
+	identifier []byte,
+	destination []byte,
+	tokenID []byte,
+	nonce uint64,
+	value *big.Int,
+) {
+	nonceBig := big.NewInt(0).SetUint64(nonce)
+
+	logEntry := &vmcommon.LogEntry{
+		Identifier: identifier,
+		Address:    destination,
+		Topics:     [][]byte{vmInput.CallerAddr, tokenID, nonceBig.Bytes(), value.Bytes()},
```

Comment:
you should not change the order of the topics because we and all the integration rely on this order. 
Topics[0] -- token identifier
Topics[1] -- nonce
Topics[2] - value